src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -0,0 +1,24 @@
+import 'isomorphic-fetch';
+
+const fetchMarkup = async url =>
+  fetch(url, { mode: 'no-cors' })
+    .then(html => {
+      return html.text().then(text => {
+        return text;
+      });
+    })
+    .catch(e => console.error(`HTTP Error: "${e}"`));
+
+const convertInclude = async ({ tile, href, platform, url }) => {
+  return {
+    type: 'include',
+    model: {
+      type: platform,
+      href,
+      tile,
+      html: url ? await fetchMarkup(url) : null,
```

Comment:
Could we drop a comment here indicating we have added url temporarily to allow a full E2E implementation including a fetch? Just so we don't get the impression url is part of the Ares response when it isn't. Should we create an issue to point this to our new public routes when the mozart issues have been completed? We could even reference it in the commment?

Reply:
@andrewscfc ideally the logic in this file should be part of https://github.com/bbc/simorgh/issues/5762 and you're right it just allows us to simulate the whole implementation.

> Should we create an issue to point this to our new public routes when the mozart issues have been completed
I totally agree with this and also referencing it in a comment. 
**This file would be pruned from this PR once it gets past review **

Reply:
> This file would be pruned from this PR once it gets past review 
Which file are you referring to here @rhenshaw56 ? I'd personally be happy leaving this as it is for the merge to latest, it works and will not be used in live. As long as we clean up in the follow-up issue.

Sorry if I'm misunderstanding though

Reply:
+1 with cleaning up in the following issue.